css/style.css

```
@@ -553,3 +590,104 @@ a {
 .madeby-link:active {
   color: var(--basic-red);
 }
+
+/* CATALOG PAGE STYLES */
+.catalog-product-title {
+  background-color: var(--alt-title-background);
+  color: var(--alt-grey);
+}
+
+.bread-crumbs-link {
+  font-family: "PT Sans", "Arial", sans-serif;
+  font-size: 13px;
+  font-weight: 400;
+  line-height: 18px;
+  color: var(--black);
+  text-transform: uppercase;
+}
+
+.product-filter,
+.product-sort {
+  background-color: var(--options-background);
+}
+
+.product-sort {
+  font-family: "PT Sans", "Arial", sans-serif;
+}
+
+.product-filter-title {
+  display: none;
+}
+
+.product-sort-title {
+  font-size: 13px;
+  font-weight: 400;
+  line-height: 18px;
+  color: var(--black);
+  text-transform: uppercase;
+}
+
+.product-sort-link {
+  font-size: 13px;
+  font-weight: 400;
+  line-height: 18px;
+  color: var(--black);
+  text-transform: uppercase;
+  opacity: 0.3;
+  border-bottom: var(--basic-red) dotted 1px;
```

обычно в таком порядке пишется
```suggestion
  border-bottom: 1px dotted var(--basic-red);
```